ui/src/components/ripple-out/RippleOutBanner.js

```diff
@@ -0,0 +1,38 @@
+import { Flex, Center, Link, Box, HStack, Text } from '@chakra-ui/react';
+
+const RippleOutBanner = () => {
+  return (
+    <Flex
+      w="100vw"
+      bgColor="#FFFFFF95"
+      h={['10vh', '12vh']}
+      p={2}
+      bgPosition="center"
+      bgSize="cover"
+      justify="center"
+      display="flex"
+    >
+      <Flex w="100vw" justify="space-around">
+        <Center>
+          <Link href="/ripple-out" _hover={{}}>
+            <HStack h="100%" spacing="1">
+              <Box>
+                <Text
+                  position="relative"
+                  fontSize={['0.7em', 'xl']}
+                  as="i"
+                  color="#182E57"
+                  fontWeight={600}
+                >
+                  CLICK HERE TO LEARN MORE ABOUT THE RIPPLE OUT CAMPAIGN !
```

Comment:
Is the banner text supposed to be like that when there's 2 lines or centered ya?
<img width="503" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/56828805/209ce9d2-1e2a-4e8d-9fe7-2140f540f7a6">

Reply:
i fixed, check again

Reply:
lgtm :)